l10n_fr_account_invoice_import_facturx/README.rst

```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Factur-X Invoices Import for France
+===================================
+
+This module implements the France-specific part of the Factur-X standard to import Factur-X invoices. In particular, it extracts the SIRET of the XML of the Factur-X invoices, which is an important information to match the supplier.
+
+Factur-X is the e-invoicing standard for France and Germany. The Factur-X specifications are available on the website of `FNFE-MPE <http://fnfe-mpe.org/factur-x/>`_.
+
+This module will be installed automatically when the modules *account_invoice_import_factur-x* and *l10n_fr_siret* are installed.
```

Comment:
that text doesn't match with the code : 

```
    "depends": [
        "account_invoice_import_facturx",
        "l10n_fr_business_document_import",
    ],
    "auto_install": True,

```

Reply:
l10n_fr_business_document_import is an auto-install module that depends on l10n_fr_siret and base_business_document_import (and account_invoice_import_facturx depends on base_business_document_import via account_invoice_import). This explains what I wrote in the README.